R/TealAppDriver.R

```diff
@@ -255,6 +248,35 @@ TealAppDriver <- R6::R6Class( # nolint: object_name.
       available_datasets[displayed_datasets_index]
     },
     #' @description
+    #' Test if CSS elements are visible on the page via Javascript call.
+    #' @param selector (`character`) CSS selector to test visibility.
+    #' A `CSS` id will return only one element if the UI is well formed.
+    #' @param content_visibility_auto,opacity_property,visibility_property (`logical`) See more information
+    #' on <https://developer.mozilla.org/en-US/docs/Web/API/Element/checkVisibility>.
+    #'
+    #' @return Logical vector with all occurrences of the selector.
+    test_visibility = function(selector,
```

Comment:
Let's stick to convention where `is` suggests logical output. `test` is not very specific and can mean many things.

```suggestion
    is_visible = function(selector,
```

Reply:
like the idea of `is_visible` or `check_visibility` but `is_visible` is more direct